R/input-checks.R

```diff
@@ -0,0 +1,53 @@
+
+#' Find missing input pieces.
+#'
+#' find_tests_without_reqs()` returns IDs from the test results that are not
+#' associated with a requirement. `find_reqs_without_tests()` goes in the
```

Comment:
I think it might be clearer to call this `find_reqs_with_missing_tests()` or something. The current name makes me think it will return a list of req's that don't have any test ID's attached to them. That might also be useful in theory, but I isn't what this does (and I don't know that we really need that... that's probably easy enough for the user to figure out themselves without a helper)

Reply:
All right, will go with that.  Thanks for the suggestion.